components/service/glean/src/main/java/mozilla/components/service/glean/BooleanMetricType.kt

```diff
@@ -46,6 +46,7 @@ data class BooleanMetricType(
             return
         }
 
+        @Suppress("EXPERIMENTAL_API_USAGE")
```

Comment:
Is there any advantage/disadvantage to go with the IDE's suggestion to use either `@UseExperimental(ObsoleteCoroutinesApi::class)` or `@ObsoleteCoroutinesApi` rather than just suppressing the warning?  This applies to all of the places that this is used.

Reply:
I started with that.  Unfortunately, those flags are "viral", in that all callers of these methods would also have to declare that.  Meaning, anyone using the Glean API would also have to add those tags.

Reply:
That makes sense!